Travel requests

miguelverweij opened the pull request:
| Q               | A
| --------------- | ---
| Bug fix?        | no?
| New sample?      | yes
| Related issues?  | no

#### What's in this Pull Request?

A new example for the SharePoint template Travel requests.

tecchan1107 commented:
Thank you for the very, very beautiful sample @miguelverweij 😍✨

However, there is one point that I noticed when I tried it: it is difficult to distinguish between approved data and unapproved data. Is it possible for you to fix it so that it is distinguishable?

tecchan1107 commented:
Thank you for the very, very beautiful sample @miguelverweij 😍✨

However, there were two things that I noticed when I tried it.

- It is difficult to distinguish between approved and unapproved data.
- I wonder if it is acceptable to press the Approve button even after approval.

Is it possible to fix them? If you have any questions, please let me know. I will do my best to support you.

miguelverweij commented:
Hi @tecchan1107! I am glad you like it. My idea for this list was to filter it on the Approved column (Approved == false) to create a sort of action list. I have added that to the Readme. 

However, I liked your input and adjusted it so that now the button changes according to the Approved value.

tecchan1107 commented:
Thanks for the reply, and the update @miguelverweij 🙏✨
I understand! I will process this pull request in a few days.

miguelverweij commented:
Cool! Thanks @tecchan1107

tecchan1107 commented:
Thanks for the great sample @miguelverweij 😍✨
I processed and merged this pull request. Also, I made the following few modifications.

- Remove `__comment__` (because it is a property not defined in the schema.)
- If I used an if statement in the `action` of `customRowAction`, it did not work, so I prepared divs for each setValue and defaultClick and changed them to be hidden according to the value of Approved.
- Changed `cursor` from arrow to default (because `arrow` cannot be set to `cursor`.)
- Added a screenshot of the new List dialog to the README to make it easier to see which list template is being used.

Please let me know if you have any questions. Once again, thank you for the beautiful sample!!
